main.cpp

```diff
@@ -7,100 +7,157 @@
 using namespace std;
 
 string initMagicWord();
-void display(int count, bool win, string guesses);
+string initHiddenWord(string);

```

Comment:
You can move these forward declarations into the Hangman.h if you #include "hangman.h".